migrations/helpers/update_filtered_positives_helper.py

```diff
@@ -218,17 +218,17 @@ def update_dart_fields(config: Config, samples: List[SampleDoc]) -> bool:
         cursor = sql_server_connection.cursor()
 
         for plate_barcode, samples_in_plate in groupby_transform(
-            samples, lambda x: x[FIELD_PLATE_BARCODE], reducefunc=lambda x: list(x)
+            samples, lambda x: x[FIELD_PLATE_BARCODE], None, lambda x: list(x)
```

Comment:
```suggestion
            iterable=samples, keyfunc=lambda x: x[FIELD_PLATE_BARCODE], valuefunc=None, reducefunc=lambda x: list(x)
```

Reply:
This one has too many arguments, so maybe is more descriptive with this

Reply:
Good call.  I did try to do this when I was trying to resolve the original issue, but it didn't like the keys when I gave them then.  Maybe I used the wrong syntax.  Anyway, your version seems to work so I've changed it.  Thanks.